# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../kdelibs/kerberos_fix.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- kdelibs-3.5.7/configure.in.orig	2007-06-05 08:57:10.000000000 +1000
+++ kdelibs-3.5.7/configure.in	2007-06-05 08:58:29.000000000 +1000
@@ -1885,12 +1885,12 @@
         GSSAPI_RPATH="-R ${GSSAPI_ROOT}/lib"
       fi
     fi
-    if test -f ${GSSAPI_ROOT}/include/gssapi/gssapi.h ; then
-      gssapi_flavor=MIT
-      GSSAPI_LIBS="${GSSAPI_LIBS}-lgssapi_krb5 -lkrb5 -lk5crypto -lcom_err ${LIBRESOLV}"
-    else
+    if test -f ${GSSAPI_ROOT}/include/heim_err.h ; then
       gssapi_flavor=HEIMDAL
       GSSAPI_LIBS="${GSSAPI_LIBS}-lgssapi -lkrb5 -lasn1 -lcrypto -lroken -lcrypt ${LIBRESOLV}"
+    else
+      gssapi_flavor=MIT
+      GSSAPI_LIBS="${GSSAPI_LIBS}-lgssapi_krb5 -lkrb5 -lk5crypto -lcom_err ${LIBRESOLV}"
     fi
   fi
   if test "x$gssapi_flavor" = "xMIT" ; then
